apps/examples/testcase/ta_tc/task_manager/utc/utc_task_manager_main.c

```diff
@@ -131,9 +131,11 @@ static void test_broadcast_handler(void *user_data, void *info)
 	}
 }
 
-void free_handler(void *info)
+void exit_handler(void *info)
```

Comment:
I think you'd better to define "test" string

Reply:
done.